blueprints/ember-freestyle/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+
+  name: 'ember-freestyle',
+
+  normalizeEntityName: function() {},
+
+  afterInstall: function() {
+    var bowerPackages = [
+      { name: 'normalize.css', target: '3.0.3' },
```

Comment:
I don't think the addon should pull in normalize. Incorporating it via the addon CSS would affect other things throughout the parent app.

I used it as a quick way to make certain things behave in the ember-freestyle dummy app, but perhaps we should just remove it altogether.